command/rubycommand/rubycommand.go

```diff
@@ -152,21 +152,48 @@ func GemUpdate(gem string) ([]*command.Model, error) {
 	return cmds, nil
 }
 
+func gemInstallCommand(gem, version string, enablePrerelease bool) []string {
```

Comment:
NOTE: it is a bit strange with this function that either `version` or `enablePrerelease` can be used, but not both.

Reply:
Did not consider that version range can be used with prerelease option.